dspace-xmlui-mirage2/src/main/webapp/xsl/core/forms.xsl

```diff
@@ -265,6 +265,28 @@
     </xsl:template>
 
 
+    <xsl:template name="languageSelectBuilder">
+        <xsl:param name="name"/>
+        <xsl:param name="selected"/>
+        <select>
+            <xsl:call-template name="fieldAttributes"/>
+            <xsl:apply-templates/>
```

Comment:
Are you sure this should be here?

Reply:
The fieldAttributes are required, but the apply-templates might not be. I'll try removing it and testing.